Directory.Packages.props

```diff
@@ -64,7 +64,8 @@
     <!-- Moq to stay below v4.20 due to behavior change. need to be sure fixed -->
     <PackageVersion Include="Moq" Version="4.18.4" />
     <PackageVersion Include="MSTest" Version="3.8.3" />
-    <PackageVersion Include="NLog" Version="5.0.4" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Good catch!! It seems like your other script is still quite useful - if it can catch things like this, we may want to keep it. What do you think?